min_and_max.py

```diff
@@ -0,0 +1,16 @@
+numbers = [1, 2, '0', '300', -2.5, 'Dog', True, 0o1256, None]
+list_without_words = []

```

Comment:
No need to define it in the global scope. Please move this variable in the function.

Reply:
Done